declarations/AdopteUnMec.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "AdopteUnMec",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.adopteunmec.com/cgu/",
+      "executeClientScripts": true,
+      "select": ".tab-content"
```

Comment:
You should remove the print CTA, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/0a33e4e4d50348ad550114c83b8e58f38cbe40a1